packages/cozy-harvest-lib/src/components/OAuthForm.jsx

```diff
@@ -81,15 +81,17 @@ export class OAuthForm extends PureComponent {
 
     return initialValues ? null : (
       <>
-        <Button
-          className="u-mt-1"
-          busy={isBusy}
-          disabled={isBusy}
-          extension="full"
-          label={t('oauth.connect.label')}
-          onClick={this.handleConnect}
-        />
-        {showOAuthWindow && (!needExtraParams || extraParams) && (
+        {(!needExtraParams || extraParams) && (
+          <Button
+            className="u-mt-1"
+            busy={isBusy}
+            disabled={isBusy}
+            extension="full"
+            label={t('oauth.connect.label')}
+            onClick={this.handleConnect}
```

Comment:
I wonder if we should not display it as disabled at first, instead of hiding it ? Isn't it weird to have a modal without button ?